src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a particular contact element from a person.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ */
+public class Contact {
+    
+    public String value;
+    protected boolean isPrivate;
+    
+    public Contact() {
```

Comment:
It is possible to push the constructor to the parent class (which is Contact class in this scenario)